sway-error/src/error.rs

```diff
@@ -442,6 +444,8 @@ pub enum CompileError {
         expected: usize,
         received: usize,
     },
+    #[error("The function \"{method_name}\" was called without parenthesis.")]
```

Comment:
Can you change this to:

"The function \"{method_name}\" takes no arguments, but was called without parenthesis. Try adding ()."

Thanks!

Reply:
Changed to "The function "{method_name}" was called without parenthesis. Try adding ()."
Removed the "takes no arguments" as the method may or may not take arguments. This error is thrown when the method is called with no arguments but it may still take arguments.